package.json

```diff
@@ -87,7 +87,7 @@
     "sudo-prompt": "6.1.0",
     "trackjs": "2.3.1",
     "udif": "0.10.0",
-    "unbzip2-stream": "1.0.11",
+    "unbzip2-stream": "github:jhermsmeier/unbzip2-stream#core-streams",
```

Comment:
Can you move your fork to the `resin-io-modules` org?

Reply:
Done.

Reply:
ping @jviotti

Reply:
We're just trying to enforce every work to be done in resin orgs.